src/woofmark.js

```diff
@@ -218,7 +218,21 @@ function woofmark (textarea, options) {
       if (currentMode === 'html') {
         textarea.value = parse('parseHTML', textarea.value).trim();
       } else {
-        textarea.value = parse('parseHTML', editable).trim();
+        var regexp = /\*\*[A-Z][^*]+ \*\*/gi;
+       textarea.value = parse('parseHTML', editable).trim();
+       if (textarea.value.match(regexp)) {
+         var reg = textarea.value.match(regexp);
+        
+         for (let i = 0; i <= reg.length - 1; i++) {
+           var regexp2 = /\*\*[A-Z][^*]+ \*\*/i;
```

Comment:
we are re declarating this regex on each `for` iteration.

Reply:
regexp2 should go outside the iteration sorry. will fix this.